service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -405,14 +311,15 @@ void shouldNotMigrateCaseIfMigrationIdIsIncorrect() {
         void shouldThrowAnExceptionIfIncorrectIDForSecondBundle() {
             UUID wrongID = UUID.randomUUID();
             List<Element<AdditionalApplicationsBundle>> additionalApplications = List.of(element(UUID.randomUUID(),
+                buildAdditionalApplicationsBundle(UUID.randomUUID())), element(UUID.randomUUID(),
                 buildAdditionalApplicationsBundle(UUID.randomUUID())), element(wrongID,
                 buildAdditionalApplicationsBundle(UUID.randomUUID())));
 
             CaseDetails caseDetails = caseDetails(additionalApplications, migrationId);
 
             assertThatThrownBy(() -> postAboutToSubmitEvent(caseDetails))
                 .getRootCause()
-                .hasMessage(String.format("Migration failed on case SA21C50011: Expected " + secondBundleID
+                .hasMessage(String.format("Migration failed on case SA21C50011: Expected " + thirdBundleID
                         + " but got " + wrongID,
                     familyManNumber, additionalApplications.get(1).getId()));
         }
```

Comment:
sorry i actually missed this in first pr but this at end can be removed "familyManNumber, additionalApplications.get(1).getId()"..as it already uses the variable wrongID and family man is hardcoded in